examples/app/weather/main.py

```diff
@@ -1,3 +1,12 @@
+''' A weather chart for three cities using a csv file.
+This illustration demonstrates different interpretation of the same data
+with the distribution option.
+
+.. note::
+    This chart needs the Scipy and Pandas package to run. See
```

Comment:
Sorry I guess I missed one last one, will be good to merge after this:

```suggestion
    This example needs the Scipy and Pandas package to run. See
```

Reply:
No worries! Its also on my part since I was not consistent, I'll keep your suggestions in mind next time :)